src/test/java/com/knightuser/factories/UserFactory.java

```diff
@@ -0,0 +1,71 @@
+package com.knightuser.factories;
+
+import com.knightuser.gateway.privatedb.representation.User;
+import com.knightuser.rest.request.UserRequestDto;
+
+public class UserFactory {
+
+    public static final String USER_LOGIN = "login";
+    public static final String CHANGED_USER_LOGIN = "changedLogin";
+    public static final String USER_PASSWORD = "password";
+    public static final String CHANGED_USER_PASSWORD = "changedPassword";
+
+    private UserFactory() {
+    }
+
+    public static UserRequestDto createUserRequestDtoInstance() {
+        return UserRequestDto.builder()
+            .login(USER_LOGIN)
+            .password(USER_PASSWORD)
+            .build();
+    }
+
+    public static UserRequestDto createUserRequestDtoWithoutLoginInstance() {
+        return UserRequestDto.builder()
+            .login(null)
+            .password(USER_PASSWORD)
+            .build();
+    }
```

Comment:
Maybe, it will be better to not writing ".login(null)" at all, it will be the same here

Reply:
I suppose, that it's better to make it clear here. But I agree that it'll be the same. I'm not sure that we will benefit from reducing the amount of code in the factories.

Reply:
Ok, understood your point of view